packages/terra-pills/README.md

```diff
@@ -0,0 +1,24 @@
+# Terra Pills
+
+[![NPM version](https://badgen.net/npm/v/terra-pills)](https://www.npmjs.com/package/terra-pills)
+[![Build Status](https://badgen.net/travis/cerner/terra-framework)](https://travis-ci.com/cerner/terra-framework)
+
+The terra-pills component provides a way to either display a static text or optionally allow trigger actions to launch popovers with the desired content.
+
+- [Getting Started](#getting-started)
+- [Documentation](https://engineering.cerner.com/terra-ui/components/terra-pills/pills/pills)
+- [LICENSE](#license)
+
+## Getting Started
+
+- Install from [npmjs](https://www.npmjs.com): `npm install terra-pills`
+
+## LICENSE
+
+Copyright 2021 Cerner Innovation, Inc.
```

Comment:
```suggestion
Copyright 2021 - present Cerner Innovation, Inc.
```

Reply:
fixed in this commit 91a9d8766efc2524f8114e0d1f9332318e52b26f